### ISSUE TITLE
Ridiculously big logout icon in tray context menu

### ISSUE THREAD

**Issue by alexbakker** [qTox/qTox#2491]
As the title says:
<img src="https://impy.me/i/f7144e.png"/>

I'm using XFCE on Arch Linux.

**Comment by agilob** [comment]
hah :>

**Comment by agilob** [comment]
I will fix it when master is unfrozen.

@tux3 are we there yet?

**Comment by tux3** [comment]
@agilob As far as I can tell, new AV isn't in toxcore master yet, but as long as you don't cause ANY conflict with the newav branch, I suppose it's okay to push to master.

**Comment by agilob** [comment]
It's a one line fix I think, I'll have XFCE tomorrow on mint.

**Comment by ovalseven8** [comment]
I can't see any difference between 'Logout' and 'Exit'. Wouldn't it be useful if after clicking 'Logout' the Login window is displayed so that you can load another profile?

**Comment by agilob** [comment]
Both icons are provided from your desktop manager.

> Wouldn't it be useful if after clicking 'Logout' the Login window is displayed so that you can load another profile?

I don't understand, that's what happens now (for me).

**Comment by ovalseven8** [comment]
@agilob: No, that's not the case on my machine.

When I logout in the profile menu in qTox, everything works fine. But when I use the tray context menu, 'Logout' is the same than 'Exit'.

Perhaps because of that error that occurs?

```
*** Error in `./qtox': malloc(): memory corruption (fast): 0x0000000003db12ff ***
```

**Comment by agilob** [comment]
This is your issue: https://github.com/tux3/qTox/issues/2480 qTox crashes for you when you logout. I can't reproduce it and don't have backtrace. If you could provide me backtrace than I can fix it.

**Comment by alexbakker** [comment]
This is still an issue, please reopen.
<img src="https://impy.me/i/0747cf.png"/>

**Comment by agilob** [comment]
interesting that exit icons has normal size

**Comment by alexbakker** [comment]
The ride never ends.
<img src="https://impy.me/i/ef9f2e.png"/>

**Comment by tux3** [comment]
@Impyy Oh come on! :)

**Comment by vessd** [comment]
The same for me
Xfce, Arch
![_2015-11-07_06-22-50](https://cloud.githubusercontent.com/assets/8783865/11013308/80f45ef2-8518-11e5-8722-739dcdf25b5c.png)

**Comment by vessd** [comment]
hmm, I tried the binary version and the problem disappeared...
#1583 also solved

before that I used the version from AUR
I'm not sure about the version from the https://github.com/Tox/arch-repo-tox, because I cann't compile it (https://github.com/Tox/arch-repo-tox/issues/16), but if I resolve dependencies by my self, then come to the same result

Fonts in the binary version look worse
![_2015-11-07_06-58-18](https://cloud.githubusercontent.com/assets/8783865/11013424/6342dffe-851e-11e5-9a47-945de6c0e310.png)
![_2015-11-07_06-58-34](https://cloud.githubusercontent.com/assets/8783865/11013423/63434584-851e-11e5-8415-185cd3d5320b.png)

**Comment by agilob** [comment]
I will have mint with xfce today and test it...

**Comment by agilob** [comment]
Ok, I have xfce on Mint Ubuntu and can't reproduce it.

**Comment by vessd** [comment]
@agilob I can reproduce it with Xubuntu 15.10 in VirtualBox...

**Comment by agilob** [comment]
I've just installed xfce4 and will fix it tomorrow.

**Comment by alexbakker** [comment]
Fixed by 7189844b2ad90f21f7b55c78b1b33953306b1636
